tests/utils.py

```diff
@@ -475,6 +475,7 @@ async def completions_with_server_args(
     '''
 
     outputs = None
```

Comment:
Why do we need this?

Reply:
```
    outputs = None
    max_wait_seconds = 240 * 3  # 240 is default
    with RemoteOpenAIServer(model_name,
                            server_cli_args,
                            max_wait_seconds=max_wait_seconds) as server:
        client = server.get_async_client()
        outputs = await client.completions.create(model=model_name,
                                                  prompt=prompts,
                                                  temperature=0,
                                                  stream=False,
                                                  max_tokens=5,
                                                  logprobs=num_logprobs)
    assert outputs is not None

    return outputs
```

Here is the full code snippet for reference.

The purpose is to handle the scenario where `client.completions.create()` fails & `outputs` is not assigned. Since `outputs` gets returned, the case where outputs is not assigned needs to be handled in some way (unless we just let `return outputs` fail with an `outputs`-is-not-defined type error.) Assigning `outputs = None` allows me to detect that the completion failed with an unhandled exception.

Do you suggest a better way?

Reply:
Would it be better if I added an error message saying "completion API call failed"

Reply:
Yeah that works. Thanks.